inventory/dynamic/gcp/hosts.py

```diff
@@ -98,7 +98,7 @@
 import os
 import time
 import argparse
-import ConfigParser
+import configparser
```

Comment:
Was not sure where to add this, but the installation of ansible (only current requirement) leaves out `pycrypto>=2.6` as well as `apache-libcloud`. These may need to be added to a requirements file for this particular script.

Reply:
I am not sure how pycrypto is used but it was listed as an import error when I tried to run the script straight away. Looks like ansible grabbed a crypto package but it wasn't the same one.

Reply:
IIRC ansible doesn't use pycrypto anymore, but I'm not sure how that relates to *this script* which may be *called* by ansible but does not call it.  Either way, this requirement declaration needs to be updated or removed, so a BZ would be appropriate.

We would need a package for libcloud before a PR here, so if anything we'd need a BZ for that (particularly if you want to backport to stable branches).

As for the docs, I'm going to have to leave that up to your team to handle.

Reply:
@patrickdillon should we make separate BZs for those mentioned above ?